thrift/thrift_test.go

```diff
@@ -237,7 +237,14 @@ func TestClientHostPort(t *testing.T) {
 }
 
 func TestRegisterPostResponseCB(t *testing.T) {
-	withSetup(t, func(ctx Context, args testArgs) {
+	key := "key"
```

Comment:
nit: can we call these `ctxKey` and `ctxValue` so it's clearer in the rest of the test